src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -210,8 +210,8 @@ private[spark] class BlockManager(
       }
 
       require(file.isDirectory(), "PMem directory is required for initialization")
-      PersistentMemoryPlatform.initialize(initPath, pmemInitialSize, 0)
-      logInfo(s"Intel Optane PMem initialized with path: ${initPath}, size: ${pmemInitialSize} ")
+      PersistentMemoryPlatform.initialize(path_postfix, pmemInitialSize, 0)
```

Comment:
```file.getAbsolutePath```

Reply:
@winningsix  Yes, the absolute path of the object should be passed as a parameter